doc/classes/Tween.xml

```diff
@@ -72,6 +72,13 @@
 				[b]Note:[/b] The [Tween] will become invalid after finished, but you can call [method stop] after the step, to keep it and reset.
 			</description>
 		</method>
+		<method name="get_total_elapsed_time" qualifiers="const">
+			<return type="float" />
+			<description>
+				Returns the total time in seconds the [Tween] has been animating (i.e. time since it started, not counting pauses etc.). The time is affected by [method set_speed_scale] and [method stop] will reset it to [code]0[/code].
+				[b]Note:[/code] Due to floating point precision, the time returned after the [Tween] has finished animating may slightly differ from the expected time.
```

Comment:
The note is not perfectly accurate. The problem is basically that
```GDScript
var time = 0
func _process(delta):
    time += delta
    if time == 2:
        print("end")
```
will never be true due to delta precision. I wasn't sure how to word it.

Reply:
Did you fix this with #59571 ?

Reply:
No, it's a different thing. The fix was about value, where the final is defined. Time doesn't really have a final value, it's just counted.

Reply:
Maybe phrase it like this:
```suggestion
				[b]Note:[/code] As it results from accumulating frame deltas, the time returned after the [Tween] has finished animating will be slightly greater than the actual [Tween] duration.
```